node-js-basics-101/node_modules/negotiator/lib/language.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * negotiator
+ * Copyright(c) 2012 Isaac Z. Schlueter
+ * Copyright(c) 2014 Federico Romero
+ * Copyright(c) 2014-2015 Douglas Christopher Wilson
+ * MIT Licensed
+ */
+
+'use strict';
+
+/**
+ * Module exports.
+ * @public
+ */
+
+module.exports = preferredLanguages;
```

Comment:
__[jshint]__
* 'module' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->